backlogger.py

```diff
@@ -170,16 +170,18 @@ def render_influxdb(data):
 
             start = datetime.strptime(issue["created_on"], "%Y-%m-%dT%H:%M:%SZ")
             end = datetime.strptime(issue["updated_on"], "%Y-%m-%dT%H:%M:%SZ")
-            result[status]["leadTime"].append((end - start).total_seconds() / 3600)
+            result[status]["leadTime"].append((end - start).total_seconds())
```

Comment:
I'd avoid the repeated access `result[status]`.

Reply:
Ok. But I would do that later. For now I think there will be a conflict with #16 so I don't want to make further changes.

Reply:
Ok, I did this now in an  extra commit.